Resources/Prototypes/Reagents/narcotics.yml

```diff
@@ -165,25 +166,6 @@
         time: 16
         refresh: false
 
-- type: reagent
```

Comment:
There is no migration for reagent removal.

Reply:
> Probably way out of scope of this PR, but a more general system for translating prototypes (x to y, x to null) of any kind should be considered.

Could you leave the old reagent prototype and add a comment saying it's deprecated in favor of THC?

Reply:
That amounts to committing a TODO. If it's really that troublesome, an announcement can be made through some official channel relevant to who it would concern and a date set for merging, instead of making two PRs.